site/content/docs/5.1/components/orange-footer.md

```diff
@@ -7,31 +7,466 @@ aliases: "/docs/5.1/components/orange-footer/"
+            <li><a class="nav-link" href="#">Subcategory</a></li>
+            <li><a class="nav-link" href="#">Subcategory</a></li>
+            <li><a class="nav-link" href="#">Subcategory</a></li>
+            <li><a class="nav-link" href="#">Subcategory</a></li>
+          </ul>
+        </div>
+      </div>
+      <div>
+        <h6 class="container accordion-header" id="headingFourSmall">
+          <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseFourSmall" aria-expanded="true" aria-controls="collapseFourSmall">
+            Category
+          </button>
+        </h6>
+        <div id="collapseFourSmall" class="container accordion-collapse collapse" aria-labelledby="headingFourSmall" data-bs-parent="#accordionExampleSmall">
+          <ul class="navbar-nav">
+            <li><a class="nav-link" href="#">Subcategory</a></li>
+            <li><a class="nav-link" href="#">Subcategory</a></li>
+            <li><a class="nav-link" href="#">Subcategory</a></li>
+            <li><a class="nav-link" href="#">Subcategory</a></li>
+            <li><a class="nav-link" href="#">Subcategory</a></li>
+            <li><a class="nav-link" href="#">Subcategory</a></li>
+            <li><a class="nav-link" href="#">Subcategory</a></li>
+            <li><a class="nav-link" href="#">Subcategory</a></li>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="border-bottom border-1 border-dark"></div>
+
+  <!-- Service links -->
+  <div class="container navbar-dark footer-service">
+    <ul class="navbar-nav">
+      <li><a class="nav-link" href="#"><svg width="1.875rem" height="1.875rem" focusable="false"><use xlink:href="#locate"/></svg>Locate a store</a></li>
+      <li><a class="nav-link" href="#"><svg width="1.875rem" height="1.875rem" focusable="false"><use xlink:href="#coverage"/></svg>Coverage checker</a></li>
+      <li><a class="nav-link" href="#"><svg width="1.875rem" height="1.875rem" focusable="false"><use xlink:href="#contact"/></svg>Contact us</a></li>
+      <li><a class="nav-link" href="#"><svg width="1.875rem" height="1.875rem" focusable="false"><use xlink:href="#protection"/></svg>Child protection</a></li>
+    </ul>
+  </div>
+
+  <div class="border-bottom border-1 border-dark"></div>
+
+  <!-- Terms & Conditions -->
+  <div class="container navbar-dark footer-terms">
+    <ul class="navbar-nav">
+      <li>© Orange {{< year >}}</li>
+      <li><a class="nav-link" href="#">Terms & Conditions</a></li>
+      <li><a class="nav-link" href="#">Privacy</a></li>
+      <li><a class="nav-link" href="#">Accessibility statement</a></li>
+      <li><a class="nav-link" href="#">Cookie policy</a></li>
+    </ul>
+  </div>
+</footer>
+{{< /example >}}
+
+## Parts
+
+The orange footer parts are independent from each other. Just add the following code between each part.
+
+```html
+<div class="border-bottom border-1 border-dark"></div>
```

Comment:
It is not possible to add those 3 classes directly in each part rather than creating an extra `<div>`?

Reply:
Not sure about that with the actual implementation as the container is at the part level.